app/code/spec/Inviqa/SymfonyContainer/Model/StoreConfigCompilerPassSpec.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace spec;
+
+use Bridge\MageApp;
+use Bridge\MageStore;
+use Inviqa_SymfonyContainer_Model_StoreConfigCompilerPass;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+class Inviqa_SymfonyContainer_Model_StoreConfigCompilerPassSpec extends ObjectBehavior
+{
+    function let(MageApp $app, MageStore $mageStore)
+    {
+        $mageStore->getConfig('store/config/key')->willReturn('some_value');
```

Comment:
not a constant as this is just a spec, but certainly can be a private variable for the spec.